server/controller/authController.js

```diff
@@ -0,0 +1,89 @@
+import jwt from "jsonwebtoken"
+import usersModel from "../model/usersModel"
+import bcrypt from "bcryptjs"
+
+class AuthController{
+    signup(req, res){
+        const {firstname, lastname, othername, email, phoneNumber} = req.body;
+        const unhashed_pass = req.body.password
+        const password = bcrypt.hashSync(unhashed_pass, bcrypt.genSaltSync(8))
+        const isAdmin = false
+        const values = [firstname, lastname, othername, email, password, phoneNumber, isAdmin]
+
+        const promis = usersModel.createUser(values)
```

Comment:
Rather than name this 'promis', Use a more befitting name, 'user', to represent the user model.

Reply:
You are yet to attend to this. do same for all cases where you used 'promis' as a variable name.